modules/scanning-images.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images.adoc
+:_module-type: CONCEPT
```

Comment:
Does this attribute declaration need to be unset at the end of the module? The same applies to the other `:_module-type` references in this PR.

Reply:
Is a requirement for publishing on PV2.